mods/parsoid.js

```diff
@@ -8,6 +8,7 @@ var P = require('bluebird');
 var URI = require('swagger-router').URI;
 var uuid   = require('cassandra-uuid').TimeUuid;
 var rbUtil = require('../lib/rbUtil');
+var Template = require('../lib/reqTemplate');
```

Comment:
Could we keep this out of the modules, and perform the Template wrapping in the module loading code in the router instead? I think that would keep the interface cleaner.